Makefile

```diff
@@ -78,3 +78,7 @@ storybook-deploy:
 .PHONY: release-major release-minor release-patch
 release-major release-minor release-patch: build
 	@$(XYZ) --increment $(@:release-%=%) --branch $(RELEASE_BRANCH)
+
+.PHONY: release-major-beta release-minor-beta release-patch-beta
+release-major-beta release-minor-beta release-patch-beta: build
+	@$(XYZ) --increment $(@:release-%=%) --branch $(RELEASE_BRANCH) --prerelease-label beta
```

Comment:
Did you use a locally installed version of `xyz`? It looks like the version in package.json does not support `--prerelease-label` (this command errors out for me). Newer versions of xyz have built-in support for preminor, prepatch, prerelease increments which could allow a more generic prerelease command. 

would appreciate either bumping `xyz` so this command actually works as written, or optionally making a more generic `make prerelease-*` command that could accept any prerelease label (e.g. `alpha` `rc` etc)

Reply:
I actually had not run this/released it yet :) Updated the version and I think simplified the Makefile for it to work correctly (so we will first run `make release-prerelease`)